src/System.IO.Pipes/src/Microsoft/Win32/SafeHandles/SafePipeHandle.Windows.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Security;
+
+namespace Microsoft.Win32.SafeHandles
+{
+    public sealed partial class SafePipeHandle : SafeHandle
+    {
+        protected override bool ReleaseHandle()
+        {
+            return Interop.mincore.CloseHandle(handle);
+        }
+
+        public override bool IsInvalid
+        {
+            [SecurityCritical]
+            get { return handle == new IntPtr(0) || handle == new IntPtr(-1); }
```

Comment:
Sure, good suggestion.  This is just the code that was already there, but I'll change it to use IntPtr.Zero instead.